### PR TITLE
remove results when null in search view (#421)

### PR DESCRIPTION
**Pull request by imGok**
searchvewwe

### CODE REVIEW

**Comment by hoangdat** [comment]
How about the UI whenever no result matched? @imGok

File: lib/presentation/view/common/common_view.dart [hunk 1]
```diff
@@ -44,6 +44,10 @@ class CommonView {
   }
 
   Widget buildResultCountRow(BuildContext context, int count) {
+    if (count == 0) {
```
Comment (huynguyennovem) [inline lib/presentation/view/common/common_view.dart:47]: You updated here but haven't used this function in the screens (my space, received share, shared space). 
If you use this, you don't need to check the length of state's list more :)

Comment (imGok) [reply lib/presentation/view/common/common_view.dart:47]: yh u are right i'll update other too

Comment (imGok) [reply lib/presentation/view/common/common_view.dart:47]: > How about the UI whenever no result matched? @imGok

image

**Comment by imGok** [comment]
> How about the UI whenever no result matched? @imGok

image

**Comment by imGok** [comment]
@hoangdat rdy to go

**Review by huynguyennovem — APPROVED** [review]
/lgtm

**Pull request merged**